etl/steps/data/garden/countries/2023-09-29/cow_ssm.countries.json

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
There are 208 unused values in mapping. Is that a problem?

Reply:
That's expected, table `cow_ssm_majors` just has a very small list of countries.